mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -348,7 +348,7 @@ class _DashboardViewState extends State<DashboardView>
     );
   }
 
-  void _refresh({bool refreshMap = true}) {
+  Future<void> _refresh({bool refreshMap = true}) async {
```

Comment:
@NoahNsimbe I noticed this dart metric warning. Does removing the async cause errors?

Reply:
Thanks @BenjaminSsempala  @Mnoble-19 I have resolved it.